src/test/java/com/github/jreddit/action/SubmitActionsTest.java

```diff
@@ -116,6 +121,28 @@ public void testCommentUserRequired() {
         assertEquals(errorOutput.toString().contains("User submission failed: please login first."), true);
         assertFalse(res);
     }
+    
+    /**
+     * Test commenting within 10 minutes from last comment.
+     */
+    @Test
+    public void testCommentTenMinutesLimit() {
+        // Stub REST client methods
+        when(user.getModhash()).thenReturn(MODHASH);
+        when(user.getCookie()).thenReturn(COOKIE);
+        when(restClient.post("thing_id=" + "fullname" + "&text=" + "text"
+                + "&uh=" + user.getModhash(),
+                ApiEndpointUtils.COMMENT, user.getCookie()))
+                .thenReturn(responseTenMinutesLimit);
+        // Save System.err.println output to variable
+        ByteArrayOutputStream errorOutput = new ByteArrayOutputStream();
+        System.setErr(new PrintStream(errorOutput));
+
+        // Attempt to comment
+        boolean res = submitAction.comment("fullname", "text");
+        assertEquals(errorOutput.toString().contains("User submission failed: You need to wait before posting again"), true);
```

Comment:
You mean at line 143 ? 

I just followed the style of testCommentUserRequired() which uses assertEquals